static/js/admin/vue-permissions-editor/tests/unit/patronRoles.spec.js

```diff
@@ -1096,97 +1062,100 @@ describe('patronRoles.vue', () => {
+                }, {
+                    assignedTo: '1234',
+                    principal: 'authenticated',
+                    role: 'canViewOriginals'
+                }]
         });
-        wrapper.vm.assignedPatronRoles.set = jest.fn().mockReturnValue([{
-            assignedTo: '1234',
-            principal: 'everyone',
-            role: 'canViewAccessCopies'
-        }, {
-            assignedTo: '1234',
-            principal: 'authenticated',
-            role: 'canViewOriginals'
-        }]);
-        wrapper.find('#is-canceling').trigger('click');
+
+        await wrapper.find('#is-canceling').trigger('click');
         expect(global.confirm).toHaveBeenCalled();
     });
 
-    it("prompts the user if 'Cancel' is clicked and an embargo has been added", () => {
+    it("prompts the user if 'Cancel' is clicked and an embargo has been added", async() => {
         wrapper.setData({
             saved_details: {
                 embargo: null
             },
             embargo: '2099-12-31'
         });
-        wrapper.find('#is-canceling').trigger('click');
+        await wrapper.find('#is-canceling').trigger('click');
         expect(global.confirm).toHaveBeenCalled();
     });
 
-    it("prompts the user if 'Cancel' is clicked and an embargo has been removed", () => {
-        wrapper.setData({
+    it("prompts the user if 'Cancel' is clicked and an embargo has been removed", async() => {
+        await wrapper.setData({
             saved_details: {
                 embargo: '2099-12-31'
             },
             embargo: null
         });
-        wrapper.find('#is-canceling').trigger('click');
+        await wrapper.find('#is-canceling').trigger('click');
         expect(global.confirm).toHaveBeenCalled();
     });
 
-    it("prompts the user if 'Cancel' is clicked and a new user has been added", () => {
+    it("prompts the user if 'Cancel' is clicked and a new user has been added", async() => {
         wrapper.setData({
             saved_details: {
                 embargo: null
             },
             new_assignment_principal: 'bigGroup',
             new_assignment_role: 'canViewOriginals'
         });
-        wrapper.find('#is-canceling').trigger('click');
+        await wrapper.find('#is-canceling').trigger('click');
         expect(global.confirm).toHaveBeenCalled();
     });
 
     it("Updates 'cancel' button text if there are unsaved changes", async () => {
-        //stubDataLoad();
-        wrapper.setData({
-            saved_details: {}
-        })
-
-        let btn = wrapper.find('#is-canceling');
-        expect(btn.text()).toBe('Close');
+        await wrapper.setData({
+            saved_details: null
+        });
+        expect(wrapper.find('#is-canceling').text()).toBe('Close');
 
-        wrapper.find('#user_type_patron').setChecked();
-        await wrapper.vm.$nextTick();
-        expect(btn.text()).toBe('Cancel');
+        await wrapper.setData({
```

Comment:
Did the previous method of causing changes (checking a box) not work anymore?

Reply:
I had mixed results with filling in forms via the updated Vue test utils. `setChecked()` has been removed. Now it's `setValue(<your value here>)`. It didn't always seem to work.